packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -457,6 +457,15 @@ export interface IContainerRuntimeOptions {
      * @experimental Not ready for use.
      */
     readonly compressionOptions?: ICompressionRuntimeOptions;
+    /**
+     * If specified, when in FlushMode.TurnBased, if the size of the ops between JS turns exceeds this value,
+     * an error will be thrown and the container will close.
+     *
+     * If unspecified, the limit is 950 * 1024.
+     *
+     * 'Infinity' will disable any limit.
+     */
+    readonly maxBatchSize?: number;
```

Comment:
please mark is as @deprecated or @experimental - we want to ensure we can easily remove it once we propagate properly driver policy

Reply:
can you also add the unit to the name, is it a bytes, count, etc?